mantidproject/fitbenchmarking
# Inverted x and y

#### Description of Work

Fixes #80 
Fixes issue in which the nist data was inverted (x was y and y was x).

#### Testing Instructions

1. Run `example_runScripts.py` in the mantidpython terminal.
2. Check the nist tables and the visual display pages of the nist problems.
